### ISSUE TITLE
Placemark problem

### ISSUE THREAD

**Issue by carmas123** [Baseflow/flutter-geolocator#499]
Hi, I try to upgrade the library to the last version, but with the 6.0.0+1 version I'm unable to use the Placemark class. I got always the dart does not recognize this class.

Please have any solution to that?

It work well with ^5.3.2+2 version.

**Comment by mvanbeusekom** [comment]
@carmas123 as mentioned in the [README.md](https://github.com/Baseflow/flutter-geolocator#features) (see the part markt as "IMPORTANT:") and the [CHANGELOG](https://github.com/Baseflow/flutter-geolocator/blob/develop/geolocator/CHANGELOG.md#600) I have removed all geocoding features from the geolocator plugin and moved them into its own plugin called "[geocoding](https://pub.dev/packages/geocoding)". This is done so I can provide better support on both plugins and hopefully extend the geocoding plugin with additional features.

To access the geocoding features simply add `geocoding: ^1.0.2` to your `pubspec.yaml` and use it as follows:

``` dart
import 'package:geocoding/geocoding.dart';

List<Placemark> placemarks = await placemarkFromCoordinates(52.2165157, 6.9437819);
List<Location> locations = await locationFromAddress("Gronausestraat 710, Enschede");
```

**Comment by carmas123** [comment]
Yes man I've done, but without any changes

**Comment by mvanbeusekom** [comment]
maybe try importing the `geocoding_platform_interface` in the Dart files where you need the geocoding features:

``` dart
import 'package:geocoding/geocoding.dart';
import 'package:geocoding_platform_interface/geocoding_platform_interface.dart';

List<Placemark> placemarks = await placemarkFromCoordinates(52.2165157, 6.9437819);
List<Location> locations = await locationFromAddress("Gronausestraat 710, Enschede");
```

I think we have a small issue in the geocoding were we don't export the `Placemark` class correctly to the `geocoding` plugin. I will roll out a fix today. I have also created a new [issue](https://github.com/Baseflow/flutter-geocoding/issues/6) in the geocoding plugin to track the problem.

**Comment by carmas123** [comment]
I've done also this test, including also geocoding_platform_interface into the pubspec

**Comment by mvanbeusekom** [comment]
You don't need to include it in your `pubspec.yaml` since it will be installed as a dependency of the geocoding package. You do need to add the extra `import` statement in the code (.dart) file where you need access to the `Placemark` class (see also this [issue](https://github.com/Baseflow/flutter-geocoding/issues/4))

**Comment by mvanbeusekom** [comment]
I have just released version 1.0.3 of the geocoding plugin which should solve this issue. Maybe you can also run a `flutter clean` just to make sure some caching still resides which thinks the `Placemark` class still belongs to the geolocator plugin.
